src/components/capture/CatalogEditor.tsx

```diff
@@ -1,22 +1,24 @@
 import { Paper, Typography } from '@mui/material';
-import DraftEditor from 'components/draft/editor';
-import useEditorStore, {
-    editorStoreSelectors,
-} from 'components/draft/editor/Store';
+import DraftSpecEditor from 'components/editor/DraftSpec';
+import { EditorStoreState, useZustandStore } from 'components/editor/Store';
+import { DraftSpecQuery } from 'hooks/useDraftSpecs';
 import { FormattedMessage } from 'react-intl';
 
 function NewCaptureEditor() {
-    const draftId = useEditorStore(editorStoreSelectors.draftId);
+    const id = useZustandStore<
+        EditorStoreState<DraftSpecQuery>,
+        EditorStoreState<DraftSpecQuery>['id']
+    >((state) => state.id);
```

Comment:
Need help not making this verbose TypeScripting. The `useZustandStore` could use work to help make this typing better. I just hit a wall and decided to brute force it.

I think this TODO getting done would help us out:
https://github.com/pmndrs/zustand/blob/main/src/react.ts#L27